Countries Info/ViewControllers/CountriesListViewController.swift

```diff
@@ -25,6 +25,12 @@ class CountriesListViewController: UIViewController {
         configureObservers()
     }
     
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        let countryDetailsVC = segue.destination as? CountryDetailsViewController
```

Comment:
Let's create this controller in the same way we did with countries list controller - using swinject, to follow the same pattern.